scripts/comparison/compare_results.py

```diff
@@ -76,35 +83,64 @@ def getSuiteData(testSuite):
     return tests
 
 
-def getTestData(xmlPath):
+def get_metadata_from_ini(iniPath):
+    entries = []
+    try:
+        config = configparser.ConfigParser()
+        config.read(iniPath)
+    except Exception:
+        print('Cannot read ini file')
+
+    if (config['METADATA']):
+        for key in config['METADATA']:
+            configEntry = dict()
+            configEntry['name'] = key
+            configEntry['value'] = config['METADATA'][key]
+            entries.append(configEntry)
+    return entries
+
+
+def get_test_data(xmlPath):
     tests = []
     try:
         tree = ET.parse(xmlPath)
         root = tree.getroot()
     except ET.ParseError:
-        root = getFixedXml(xmlPath)
+        root = get_fixed_xml(xmlPath)
 
     if (root.tag == 'testsuites'):
         for testSuite in root:
-            tests += getSuiteData(testSuite)
+            tests += get_suite_data(testSuite)
     else:
-        tests = getSuiteData(root)
+        tests = get_suite_data(root)
 
-    tests = cleanDuplicates(tests)
+    tests = clean_duplicates(tests)
     return tests
 
 
 if __name__ == "__main__":
-    patchedResultsPath, unpatchedResultsPath, outputFile = getParameters(sys.argv[1:])
+    patchedResultsPath, unpatchedResultsPath, iniFile, outputFile = get_parameters(sys.argv[1:])
     scriptPath = os.path.dirname(os.path.realpath(__file__))
 
-    newHtml = htmlReport()
-
-    patchedResults = getTestData(patchedResultsPath)
-    unpatchedResults = getTestData(unpatchedResultsPath)
-
-    newSection = htmlReportSection(
-            wrapper=['<table cellpadding="0" cellspacing="0">', '</table>'])
+    newHtml = HtmlReport()
+
+    if (iniFile):
```

Comment:
no need for (iniFile) -- and keep stuff consistent with variable_names and not variableNames